docs/main.md

```diff
@@ -0,0 +1,54 @@
+@page react-view-models react-view-models
+@parent can-ecosystem
+@description `react-view-models` connects observable view-models to [React](https://facebook.github.io/react/) [presentational components](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0#.v9i90qbq8) to create auto-rendering [container components](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0#.v9i90qbq8).
+@package ../package.json
+
+@body
+
+## Description
+
+`react-view-models` follows the pattern popularized by [react-redux](https://github.com/reactjs/react-redux) and provides users with a [react-view-models.reactViewModel reactViewModel] function for extending [React](https://facebook.github.io/react/) Presentational Components into Container Components, by providing a `ViewModel` constructor function, which is an extended [DefineMap](./can-define/map/map.html).
+
+The `ViewModel` is an observable, and when any observable change happens to one of its properties, or if new props get set on the Container Component, some of the view-model’s properties will be sent into the connected component as props, forcing an update/render.
+
+By following the patterns established by react-redux, but avoiding the complexity of pure-functional programming, reducer composition, immutability, and the single store paradigm, `react-view-models` offers a familiar, powerful, yet far simpler solution to creating great state management and data stores for your React app.
```

Comment:
I think the last bit of this sentence would be more effective if it were something like “solution to managing state and data in your React app.”